pkg/object/address.go

```diff
@@ -59,3 +68,34 @@ func (a *Address) GetObjectID() *ID {
 func (a *Address) SetObjectID(id *ID) {
 	(*refs.Address)(a).SetObjectID(id.ToV2())
 }
+
+// Parse converts base58 string representation into Address.
+func (a *Address) Parse(s string) error {
+	var (
+		err   error
+		oid   = NewID()
+		cid   = container.NewID()
+		parts = strings.Split(s, "/")
```

Comment:
Please add `const AddressSeparator = "/"` and use it there and in `String()`

Reply:
yeah, it makes sense

Reply:
decided to use `addressSeparator`

Reply:
fixed